t/endpoint.t

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env perl
+
+use 5.028;
+use warnings;
+use experimental qw(signatures);
+
+use Test::Lib;
+use Test::PromAlertProxy;
+use Test::More;
+use Test::Time time => Test::PromAlertProxy->now;
+
+use IO::Async::Loop;
+
+use Plack::Test;
+use HTTP::Request::Common;
+use JSON::MaybeXS;
+
+use PromAlertProxy::Hub;
+
+use PromAlertProxy::Logger '$Logger' => { init => {
```

Comment:
I initialise it because I don't know where it will go, and imagined the test runs might be spewing crap out into syslog or somewhere, which would be rude.

But yeah, shedding boilerplate is good.